jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/RuntimeDataServiceImplTest.java

```diff
@@ -269,10 +269,10 @@ public void testGetProcessesSortByProcessName() {
     	List<String> expectedProcessIds = new ArrayList<String>();
 
     	expectedProcessIds.add("AdHoc SubProcess");
+		expectedProcessIds.add("humanTaskSample");
```

Comment:
formatting?

Reply:
Ok I will check it.

Reply:
I find it is correct in my local.

Reply:
From the diff, there is a clear difference between the lines above and below. For consistency, just keep the same.